vm-setup/roles/v1aX_integration_test/tasks/remediation.yml

```diff
@@ -71,30 +85,30 @@
     become: yes
     become_user: root
 
-  - name: Power off "{{ WORKER_NODE }}"
-    shell: |
-       kubectl annotate bmh "{{ WORKER_NODE }}" -n "{{ NAMESPACE }}" reboot.metal3.io/poweroff=
+  # Power off a single worker node
```

Comment:
this is a bit misleading because it's apower cycle, i.e. it is turned on after being turned off

Reply:
Yes true. Fixed.